src/transformers/processing_utils.py

```diff
@@ -120,7 +118,6 @@ class TextKwargs(TypedDict, total=False):
     return_offsets_mapping: Optional[bool]
     return_length: Optional[bool]
     verbose: Optional[bool]
-    padding_side: Optional[str]
```

Comment:
I removed padding_side in this PR as it should be one of the first uniformization PR to get merged. This shouldn't break anything (see https://github.com/huggingface/transformers/pull/32544#discussion_r1722505343)

Reply:
I believe it will break things in the sense that people who were previously passing in `padding_side` to the processor (even if it had no effect) would now experience an error, I think?

Reply:
OK, I see. More generally, we might want to think about being able to control the padding side when calling the tokenizer, as this is something we'd like to be able to handle for e.g. llava and more generally, but this is something for a wider discussion and future PRs cc @zucchini-nlp 

In this case, if `padding_side` can't be accepted by fast tokenizers then yes, I think we should remove it here. I'd do this in a separate PR as it might affect other processors which used `TextKwargs` and so it would be good to introduce as an atomic change we can easily test, isolate and revert if needed

Reply:
Seems like we are welcome to acceptring `padding_side` as kwargs in tokenozers (https://github.com/huggingface/transformers/issues/30447) but I couldn't find a PR for that. I totally agree that it's a nice feature to have and looks easy to implement. I can work in it next week so that we son't have to drop the kwarg in processors. The only thing is that it will be kinda BC breaking, because earlier we ignored the kwarg and now we'll use it to pad on the correct side. WDYT? @amyeroberts

Reply:
@zucchini-nlp Sounds good to me! Even if it's breaking, I think it's breaking in the right way: correcting a surprising behaviour

Reply:
Added back `padding_side` to this PR. Waiting on `padding_side` to be accepted as kwargs in tokenizers before merging this PR then. Could you ping this PR or me when you open a PR @zucchini-nlp ? Thanks!

Reply:
Here is it https://github.com/huggingface/transformers/pull/33385